### DISCUSSION TITLE
How do you track salary savings transactions each month?

### DISCUSSION THREAD

**Discussion by pedrom34** [firefly-iii/firefly-iii#5642] [Firefly III]
James, 
I use firefly-iii for a couple of years now, and I still use the method you described on this page: https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/personal-finances/

But the "Balance" block on the home page only works well if my income is included in the current month.
However, I am paid around the 25th. So I transfer my entire salary to my savings account immediately, and I transfer the total amount of the budgets defined in Firefly-III on the 1st of the following month. Exactly as it says on the link.

My question: how should this appear in Firefly-III? Because if I show it as it is on my bank record, the "Balance" block does not show the real information.
Right now, I am indicating in Firefly-III that my salary is paid on the 1st of each month directly to my savings account. But this does not seem to be compatible with piggy banks...

Thanks for your answers and feedback!

**Comment by JC5** [comment]
The "balance" block shows you the sum of the money spent + money earned, nothing else. In the new layout im working on ill probably drop the block entirely.

**Reply by pedrom34** [reply]
Okay! Well, I will adapt :)
But my question remains: is my method of recording payroll savings transactions consistent with the spirit of Firefly-III? Or should I stick as closely as possible to my bank record?

**Reply by JC5** [reply]
I make sure the money on my bank account (so the salary) is also moved to my savings account. That way they're in sync again.
